Source/ViewControllers/BrickViewController.swift

```diff
@@ -8,6 +8,15 @@
 
 import UIKit
 
+#if os(iOS)
+/**
+ Conform to this delegate on your preview view controller in order to enable UIKit Pop.
+ */
+public protocol BrickViewControllerPreviewing: class {
+    weak var sourceBrick: Brick? { get set }
```

Comment:
The `weak` designation may be a mistake, I was trying to avoid a retain cycle with the brick that owned the delegate that created the view controller. However, making it optional allows for a couple points of flexibility:

* You can instantiate new instances of the view controller that conforms to this in ways other than peek/pop
* The implementation of peek allows for scenarios where you only want to peek and not pop, and by making setting this property a requirement of initialization, that gets muddled some
* The developer does not need to create an initializer for setting this value immediately at initialization